### DISCUSSION TITLE
Carousel as part of NS8

### DISCUSSION THREAD

**Discussion by darko-bw** [NativeScript/rfcs#16] [1. Ideas & Discussions]
I know it is the most requested functionality in NS. 

We have a couple of plugins but their functionality limitation is massive compare to ReactNative and Flutter options. 

Is there any chance we can have it implemented to NS8?

**Comment by farfromrefug** [comment]
I dont think this should be part of N. however i agree having a great carousel implementation could be added as a plugin
Can you point to reactnative implementations?

**Reply by darko-bw** [reply]
Maybe you are absolutely right. And it should be a standalone plugin.

The reactnative one can be found here: https://github.com/archriss/react-native-snap-carousel

**Reply by farfromrefug** [reply]
Ok this is not a native implementation per say. And i dont think we should use it with N. We should instead find great native implementations and wrap them in N.

**Reply by vallemar** [reply]
Wouldn't this behavior be the same as this plugin?   https://github.com/nativescript-community/ui-pager
